partials/single-featured.php

```diff
@@ -0,0 +1,18 @@
+<?php
```

Comment:
I agree. Or even `featured-single-include` or `featured-single-layout` something like that